src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -291,6 +323,34 @@ private bool BuildDiagnosticsClient()
                     return false;
                 }
             }
+            else if (port.Length != 0)
+            {
+                ReversedDiagnosticsServer server = new ReversedDiagnosticsServer(port);
+                server.Start();
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    Console.WriteLine($"Waiting for connection on {port}");
+                    Console.WriteLine($"Start an application with the following environment variable: DOTNET_DiagnosticPorts={port},suspend");
```

Comment:
`suspend` is the default state for Diagnostic Ports
```suggestion
                    Console.WriteLine($"Start an application with the following environment variable: DOTNET_DiagnosticPorts={port};");
```